crates/relayer-cli/src/commands/tx/channel.rs

```diff
@@ -108,6 +113,16 @@ pub struct TxChanOpenInitCmd {
         help = "The channel ordering, valid options 'unordered' (default) and 'ordered'"
     )]
     order: Ordering,
+
+    #[clap(
+        long = "connection-hops",
+        value_name = "CONNECTION_HOPS",
+        use_delimiter = true,
+        value_delimiter = ',',
+        help = "The comma separated identifiers of the intermediate connections between /
+        a source and destination chain for a multi-hop channel (optional)"
+    )]
+    conn_hop_ids: Option<Vec<ConnectionId>>,
```

Comment:
I wonder if we should re-use the `ConnectionIds` type you have introduced here instead of `Vec<ConnectionId>`?

```suggestion
        help = "A list of identifiers of the intermediate connections between /
        a source and destination chain for a multi-hop channel, separated by slashes, eg. connection-1/connection-0 (optional)"
    )]
    conn_hop_ids: Option<ConnectionIds>,
```